liberapay/utils/__init__.py

```diff
@@ -109,6 +109,8 @@ def get_community(state, restrict=False):
 
 def b64decode_s(s, **kw):
     udecode = lambda a: a.decode('utf8')
+    if not isinstance(s, bytes):
+        s = s.encode('utf8')
```

Comment:
I only meant that the `s` string here isn't supposed to contain any non-ascii char, so the code would be clearer if we used `'ascii'` instead of `'utf8'`.